### PR TITLE
Created New Secion

### PR DESCRIPTION
**Pull request by poorvaGokhale**
Created new Section: Managing Contracts
It covers aspect how to manage Contracts Driven Development with SpecMatic in Organisation where multiple Services are dependent

### CODE REVIEW

File: documentation/Versioning.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ That would not be backward compatible with version 1, and hence this would be ve
 
 ## Contract Namespaces And File System Structure
 
-We share contracts by committing them to a git repository.
+We share contracts by committing them to a git repository. Refer [Manage Your Contracts](/documentation/manage_contracts.html)
 
 Here's a sample git repository that contains the math contract:
```
Comment (joelrosario) [inline documentation/Versioning.md:75]: If we're keeping this here, can you correct the names of the examples as well, e.g. math_1.spec should be api_math_v1.spec.

File: documentation/Versioning.md [hunk 2]
```diff
@@ -84,7 +84,6 @@ Here's a sample git repository that contains the math contract:
 
 - The path ./run/specmatic/examples acts as a namespace, much like a dot separated package name.
 - Contract file names can begin with any text (no spaces)
-    - If there is a version number, then they must include version number at the end of the name before ".spec" extension.
```
Comment (joelrosario) [inline documentation/Versioning.md:87]: Should keep this line, or remove the entire section.

Comment (poorvaGokhale) [reply documentation/Versioning.md:87]: @joelrosario  feel Entire Section removed as its present in detail in Manage Contracts

File: documentation/Versioning.md (hunk 1 above)
Comment (poorvaGokhale) [reply documentation/Versioning.md:75]: Will correct file Name Versioning here.

**Comment by poorvaGokhale** [comment]
Review Comments Incorporated

**Comment by joelrosario** [comment]
Merged directly

**Pull request closed without merging**